recipes/watchme/meta.yaml

```diff
@@ -0,0 +1,94 @@
+{% set name = "watchme" %}
+{% set version = "0.0.26" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5c8b27a16faa80610dfad115970532deaaf2aeaac24a72ee679be0d5a83232eb
+
+build:
+  number: 0
+  entry_points:
+    - watchme=watchme.client:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3
+    - setuptools
+  run:
+    - python-crontab>=2.3.6
```

Comment:
These all need to have a space before the >= (but not after, just a single space between the package name and version constraint)